oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/AbstractKMeansEvalIT.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2015, Cloudera and Intel, Inc. All Rights Reserved.
+ *
+ * Cloudera, Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"). You may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for
+ * the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package com.cloudera.oryx.app.mllib.kmeans;
+
+import java.io.Serializable;
+import java.util.List;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.function.Function;
+import org.apache.spark.mllib.linalg.Vector;
+import org.apache.spark.mllib.linalg.Vectors;
+
+import com.cloudera.oryx.lambda.AbstractSparkIT;
+
+public class AbstractKMeansEvalIT extends AbstractSparkIT implements Serializable {
+
+  protected JavaRDD<Vector> getRddOfVectors(List<double[]> pointsList) {
+    return getJavaSparkContext().parallelize(pointsList).map(
+        new Function<double[], Vector>() {
```

Comment:
This is likely the problem with `Serializable`. This anonymous inner class is non-static. Break it out as a private static member